src/test/java/org/assertj/core/api/date/DateAssert_with_string_based_date_representation_Test.java

```diff
@@ -73,12 +75,22 @@ public void date_assertion_should_support_timestamp_string_representation() thro
     assertThat(date).isEqualTo(timestampAsString);
   }
 
+  @Test
+  public void date_assertion_should_support_date_with_utc_time_zone_string_representation() throws ParseException {
+    SimpleDateFormat isoFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+    isoFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+    Date date = isoFormat.parse("2003-04-26T00:00:00");
+
+    assertThat(date).isEqualTo("2003-04-26T00:00:00+00:00");
+  }
+
```

Comment:
can we add a test with the same point in time expressed in different timezones?

Reply:
Done